hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/HealthTransformations.scala

```diff
@@ -42,173 +38,241 @@ object HealthTransformations {
+    HealthCondition("infectious", "rmsf", None, Some(("dx_rmsf", "dx_infectious"))),
+    HealthCondition("infectious", "rndworm", None, Some(("dx_rndworm", "dx_infectious"))),
+    HealthCondition("infectious", "slmosis", None, Some(("dx_slmosis", "dx_infectious"))),
+    HealthCondition("infectious", "slmpois", None, Some(("dx_slmpois", "dx_infectious"))),
+    HealthCondition("infectious", "tpworm", None, Some(("dx_tpworm", "dx_infectious"))),
+    HealthCondition("infectious", "toxop", None, Some(("dx_toxop", "dx_infectious"))),
+    HealthCondition("infectious", "tular", None, Some(("dx_tular", "dx_infectious"))),
+    HealthCondition("infectious", "whpworm", None, Some(("dx_whpworm", "dx_infectious"))),
+    HealthCondition("infectious", "infect_other", None, Some(("dx_infect_other", "dx_infectious")))
+  )
 
-  //////////////////////////////////////
-  //// specific condition type maps ////
-  //////////////////////////////////////
+  val conditionTypes: Map[String, Long] = Map(
+    "eye" -> 0,
+    "infectious" -> 1 // prolly need to make this way higher to fit in order correctly
+  )
 
-  // eye disorders and diseases
-  val eye_cg_disorders: Map[String, Long] = Map(
+  val conditions: Map[String, Long] = Map(
+    // eye disorders and diseases
     "blind" -> 0,
     "cat" -> 1,
     "glauc" -> 2,
     "kcs" -> 3,
     "ppm" -> 4,
     "miss" -> 5,
-    "other" -> 98
-  )
-
-  val eye_diseases: Map[String, Long] = Map(
-    "cat" -> 0,
-    "blind" -> 1,
-    "ce" -> 2,
-    "conj" -> 3,
-    "cu" -> 4,
-    "dist" -> 5,
-    "kcs" -> 6,
-    "ectrop" -> 7,
-    "entrop" -> 8,
-    "glauc" -> 9,
-    "ilp" -> 10,
-    "ic" -> 11,
-    "jcat" -> 12,
-    "ns" -> 13,
-    "pu" -> 14,
-    "pra" -> 15,
-    "rd" -> 16,
-    "uvei" -> 17,
-    "eye_other" -> 98
-  )
-
-  val infectious_diseases: Map[String, Long] = Map(
-    "anaplasmosis" -> 0,
-    "asperg" -> 1,
-    "babesio" -> 2,
-    "blastomy" -> 3,
-    "bordetella" -> 4,
-    "brucellosis" -> 5,
-    "campylo" -> 6,
-    "chagas" -> 7,
-    "ccdia" -> 8,
-    "ccdio" -> 9,
-    "crypto" -> 10,
-    "dermato" -> 11,
-    "dstmp" -> 12,
-    "ehrlich" -> 13,
-    "fever" -> 14,
-    "gp" -> 15,
-    "giar" -> 16,
-    "granu" -> 17,
-    "hrtworm" -> 18,
-    "histo" -> 19,
-    "hepato" -> 20,
-    "hkworm" -> 21,
-    "influ" -> 22,
-    "isosp" -> 23,
-    "leish" -> 24,
-    "lepto" -> 25,
-    "lyme" -> 26,
-    "mrsa" -> 27,
-    "mycob" -> 28,
-    "parvo" -> 29,
-    "plague" -> 30,
-    "pythium" -> 31,
-    "rmsf" -> 32,
-    "rndworm" -> 33,
-    "slmosis" -> 34,
-    "slmpois" -> 35,
-    "tpworm" -> 36,
-    "toxop" -> 37,
-    "tular" -> 38,
-    "whpworm" -> 39,
-    "infect_other" -> 98
+    "ce" -> 6,
+    "conj" -> 7,
+    "cu" -> 8,
+    "dist" -> 9,
+    "ectrop" -> 10,
+    "entrop" -> 11,
+    "ilp" -> 12,
+    "ic" -> 13,
+    "jcat" -> 14,
+    "ns" -> 15,
+    "pu" -> 16,
+    "pra" -> 17,
+    "rd" -> 18,
+    "uvei" -> 19,
+    "eye_other" -> -1,
```

Comment:
I worry that using a negative number here will cause some bad downstream effect bc nothing works the way it should.

Reply:
lmao yeah good point. I'll set it to like 999 or something for now